CONTRIBUTING.md

```diff
@@ -134,26 +141,52 @@ For additional information on using the `dep` tool, see
 
 ## Patch format
 
+### General format
+
 Beside the `Signed-off-by` footer, we expect each patch to comply with the
 following format:
 
 ```
-Subsystem: One line change summary
+subsystem: One line change summary
+
+More detailed explanation of your changes (why and how)
+that can span as many lines as required.
+
+A "Fixes #XXX" comment listing the github issue this change resolves.
+This is required for the main patch in a sequence - see the examples below.
 
-More detailed explanation of your changes: Why and how.
+Signed-off-by: <contributor@foo.com>
+```
 
-See:
-    http://chris.beams.io/posts/git-commit/
+The body of the message should not just be a continuation of the
```

Comment:
Lines 161-163 suggested rewrite: The body of the message is not a continuation of the subject line and is not used to extend the subject line beyond its character limit. The subject line is a complete sentence and the body is a complete, standalone paragraph.